bigtable_rs/build.rs

```diff
@@ -90,7 +90,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         )
         .extern_path(".google.protobuf", "::prost_wkt_types")
         .file_descriptor_set_path(&descriptor_file)
-        .compile(
+        .compile_protos(
```

Comment:
Please rebase? The genrated code is also updated, I have made those change in main so you do not need to fix it here.